db/migrate/20250207061552_add_exchange_and_currency_to_securities.rb

```diff
@@ -0,0 +1,6 @@
+class AddExchangeAndCurrencyToSecurities < ActiveRecord::Migration[7.2]
+  def change
+    add_column :securities, :currency, :string
+    add_index :securities, :currency
+  end
```

Comment:
We'll want to remove this field and index.  When we search our provider for a specific security, it will return a `currency` value that we then load directly into the `Security::Price` model.

This way, we have the `price` and `currency` sitting on the same table where it is needed to construct `Money` objects.